pages/api/maas/tickets.ts

```diff
@@ -121,12 +121,12 @@ export const handler = async (
 
   // TODO: This block is for the launch campaign discount.
   // TODO: Remove after launch campaign is done.
-  const campaignStart = moment('2020-09-26 00:00')
-  const campaignEnd = moment('2020-09-28 00:00')
+  const campaignStart = moment('2020-10-03 00:00')
+  const campaignEnd = moment('2020-10-05 00:00')
```

Comment:
This block can be removed as 12e is the normal price for the ticket

Reply:
Right. Let's do that.